stories/components/data-display/resizable.stories.tsx

```diff
@@ -334,7 +334,7 @@ export const nestedResizable: Story = () => {
 export const isDisabled: Story = () => {
```

Comment:
How about modifying the title from `isDisabled` to `disabled` ?

Reply:
Fixed.
https://github.com/yamada-ui/yamada-ui/pull/3690/commits/21a77d8479e51ebefecb07e3c717cbf0557aed47